### PR TITLE
bench/ft_transfer: add db table

### PR DESCRIPTION
**Pull request by mooori**
Related to #11405.

Not sure if this directory makes sense, but I couldn't find another place where this should obviously be. I'd like to push more files with code/commands related to db setup and admin, so I think having a dedicated directory can be useful.

Proposing this table as a starting point, probably we'll have multiple iterations anyway as benchmarks are fleshed out.

### CODE REVIEW

**Review by akashin — APPROVED** [review]
We also need to track the network setup used in the benchmark - is this a single node or multiple nodes, if single node, which hardware was used for this? We can probably start with a free-form string field.

File: benchmarks/continous/db/ft_transfer_schema.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+create table ft_transfer (
+    id serial primary key,
+
+    -- Metada related to the execution of the benchmark.
```
Comment (akashin) [inline benchmarks/continous/db/ft_transfer_schema.sql:4]: Nit: Metada -> Metadata

**Comment by mooori** [comment]
I split the extra info into multiple columns, assuming it might be interesting to query/display it separately. Or is there a reason to better combine it into a single column?

**Comment by akashin** [comment]
> I split the extra info into multiple columns, assuming it might be interesting to query/display it separately. Or is there a reason to better combine it into a single column?

I think it's fine either way. Let's roll with current version and adjust as needed.

Simple text column can be more flexible and less demanding for the writer but makes some types of queries more difficult to express.

**Pull request merged**